be/src/fs/hdfs/fs_hdfs.cpp

```diff
@@ -308,24 +308,24 @@ class HdfsFileSystem : public FileSystem {
 Status HdfsFileSystem::path_exists(const std::string& path) {
     std::string namenode;
     RETURN_IF_ERROR(get_namenode_from_path(path, &namenode));
-    HdfsFsHandle handle;
-    RETURN_IF_ERROR(HdfsFsCache::instance()->get_connection(namenode, &handle, _options));
-    return _path_exists(handle.hdfs_fs, path);
+    std::shared_ptr<HdfsFsClient> hdfs_client = std::make_unique<HdfsFsClient>();
```

Comment:
why make unique ptr and assign it to shared ptr?

Reply:
I will change it.